packages/server/src/api/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Router } from 'express';
 import { PrismaClientSingleton } from '../utils';
 import { emailValidator, idValidator, userTopicsValidator } from '../validators';
-import { ZodError } from 'zod';
+import { ZodError, z } from 'zod';
```

Comment:
You did a great job but can you please add your ZOD validation to the folder `validators`. See the other validations

Reply:
Import it then use it also try to extend from the existing Zod validation. Learn from the old validation logic

Reply:
sure, thanks for the input

Reply:
Hi @prasenjeet-symon
Should I refine the existing idValidator with the "not 0" condition or should I add a new paramIdValidator?
Which one would you prefer?

Reply:
Modify the existing one since any MYSQL id cannot be 0.

Reply:
Refining existing validator would we a better choice